### DISCUSSION TITLE
Create-Pull-Request Not Found

### DISCUSSION THREAD

**Discussion by nickcdon** [community/community#31358] [Apps, API and Webhooks] [answered]
<img width="1249" alt="image" src="https://user-images.githubusercontent.com/18720809/187454729-47409bb7-6508-477b-9495-74ecfb9a1b9a.png">
<img width="652" alt="image" src="https://user-images.githubusercontent.com/18720809/187454794-6bc26286-3e4a-4910-8198-190ff4a76645.png">

**Comment by nickcdon** [comment]
<img width="761" alt="image" src="https://user-images.githubusercontent.com/18720809/187455199-ca535199-e015-4a19-92af-485b146e21a3.png">

**Comment by jfullstackdev** [comment]
do you own the repo ? you must have a write access scope

if you don't own it, you must be granted with write access or else, even if you checked all, it will not work

**Reply by nickcdon** [reply]
thanks, this is fork repo, can I use Create-Pull-Request API?

**Reply by nickcdon** [reply]
I have write access to the repo, and using List-Pull-Requests API I can get 200 response.

**Reply by nickcdon** [reply]
Thanks, I know the reason, I wrongly adjusted 'Bearer' in Authorization to 'owner'.

**Comment by nickcdon** [comment, marked as answer]
I wrongly adjusted 'Bearer' in Authorization to 'owner'.
